docs/OverseeingIssuePrLifeCycle.md

```diff
@@ -0,0 +1,36 @@
+# Overseeing Issue/PR Life Cycle
+
+## Triaging Issues
+
+Our issue tracker doubles as the message board. Therefore, it can have two types of issues:
+* Support Requests: (i.e. announcements, questions, support requests, self-intros etc.) 
+* Change Requests: Issues requiring changes (i.e. all other issues).
+ 
+When an issue comes in, 
+
+* any one can respond to it. The faster we can respond to issues, the better it is.
+* any one with sufficient privileges can add labels based on the [project's definition of each label](DefiningLabels.md).
+  * However, Project Manager (PM) should be the one applying the priority (i.e. `p.`) label 
+    for change requests (support requests do not need priority labels). This is to ensure that all issues get
+    prioritized using a uniform scale.
+
+# Assigning reviewers for PRs
+
+* A PR should have,
+  * **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)
+  * **no more than one `assignee`** : The `assignee` field can be used to indicate the main reviewer when there are 
```

Comment:
So, yeah, as long as the "reviewers" feature remains not-very-useful, I hope this restriction can be relaxed.